src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -99,14 +99,24 @@ function runEventInBatch(event) {
 function startWatchingForChangeEventIE8(target, targetID) {
   activeElement = target;
   activeElementID = targetID;
-  activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+  } else {
+    activeElement.addEventEventListener('change', manualDispatchChangeEvent, false);
```

Comment:
I think this is currently unreachable yeah, Its been a few months but I may have jsut been overzealous refactoring. Let me remove it...